sklearn/setup.py

```diff
@@ -1,13 +1,16 @@
 import os
 
 from sklearn._build_utils import maybe_cythonize_extensions
+from sklearn._build_utils.openmp_helpers import check_openmp_support
 
 
 def configuration(parent_package='', top_path=None):
     from numpy.distutils.misc_util import Configuration
     from numpy.distutils.system_info import get_info
     import numpy
 
+    with_openmp = check_openmp_support()
```

Comment:
could this be placed below right before calling `maybe_cythonize_extensions`, or do you need it here at the top for a special reason?

Reply:
no reason. Actually I moved it in `maybe_cythonize_extensions`